image/image.go

```diff
@@ -38,6 +38,9 @@ import (
 	"github.com/snapcore/snapd/snap"
 	"github.com/snapcore/snapd/snap/squashfs"
 	"github.com/snapcore/snapd/store"
+
+	// important so that the testkeys get imported
```

Comment:
they are only really imported given some build tags, this one is weird if it's needed we should organize things a bit differently